akka-cluster-sharding/src/test/java/akka/cluster/sharding/ClusterShardingTest.java

```diff
@@ -174,7 +174,7 @@ private void receiveDecrement() {
     }
 
     private void passivate() {
-      getContext().parent().tell(
+      getContext().getParent().tell(
           new ShardRegion.Passivate(PoisonPill.getInstance()), getSelf());
```

Comment:
replace with `self()`, perhaps search replace all remaining occurrences of `getSelf()`